src/ray/core_worker/experimental_mutable_object_manager.h

```diff
@@ -113,46 +114,68 @@ class ExperimentalMutableObjectManager {
   /// \param[in] object_id The ID of the object.
   Status SetError(const ObjectID &object_id);
 
+  /// Returns the unique semaphore name for the object. This name is intended to be used
```

Comment:
Should we make these methods private?

Reply:
I made GetSemaphoreName() private. I left the others public because they are called by the unit tests.

Reply:
Ah I see, you can make the unit test a friend class.

Reply:
Done.